beets/autotag/mb.py

```diff
@@ -265,6 +265,7 @@ def album_info(release):
             )
             ti.disctitle = disctitle
             ti.media = format
+            ti.alt_track_no = track['number']
```

Comment:
My understanding from looking at a handful of XML dumps is that `position` always holds an integer index of the track and the `number` holds whatever the user inputted when adding tracks. If you had an album on a double-sided CD, tape, vinyl, etc., then there's the potential of having the `number` being something like `A1`, or whatever the case may be.

So it seems to me that simply adding this data should have zero impact with sorting.